### ISSUE TITLE
parsing time "2023-04-11 20:58:43.130000": extra text: "000": Could not process SQL results

### ISSUE THREAD

**Issue by antonysouthworth-halter** [grafana/athena-datasource#447]
**What happened**:

parsing time "2023-04-11 20:58:43.130000": extra text: "000": Could not process SQL results

**What you expected to happen**:

no error

**How to reproduce it (as minimally and precisely as possible)**:

pretty sure this is because Athena only support `timestamp(6)` and is returning such data, whatever is being used for parsing datetime strings in this project not able to handle the microsecond precision

**Screenshots**

**Anything else we need to know?**:

**Environment**:
- Grafana version:
- Plugin version:
- OS Grafana is installed on:
- User OS & Browser:
- Others:

**Comment by antonysouthworth-halter** [comment]
weird, because athenadriver has a time pattern that looks like it ought to work...

https://github.com/uber/athenadriver/blob/master/go/datetime.go#L41

**Comment by antonysouthworth-halter** [comment]
local test, this works

```
package main

import (
	"fmt"
	"time"
)

func main() {
	dt, err := time.Parse("2006-01-02 15:04:05.000000", "2023-04-11 20:58:43.130000")
	if err != nil {
		panic(fmt.Errorf("unable to parse: %v", err))
	}

	fmt.Printf("%v\n", dt)
}
```

and replacing the layout with microseconds pattern causes same error

```
package main

import (
	"fmt"
	"time"
)

func main() {
	dt, err := time.Parse("2006-01-02 15:04:05.000", "2023-04-11 20:58:43.130000")
	if err != nil {
		panic(fmt.Errorf("unable to parse: %v", err))
	}

	fmt.Printf("%v\n", dt)
}
```

**Comment by antonysouthworth-halter** [comment]
We are using version 2.11.1 of this plugin, will try bump and report back. Apologies if you are getting notification spam!

**Comment by antonysouthworth-halter** [comment]
Ahhh, ok, this plugin uses 1.1.14 of uber athena driver, this has been fixed by 1.1.15

https://github.com/uber/athenadriver/releases/tag/v1.1.15

See: adds microseconds+nanoseconds precision layouts.
